### ISSUE TITLE
cmd tab key cycles skips items for directories/files completion

### ISSUE THREAD

**Issue by ogdenpm** [microsoft/terminal#7157]
yes

When running the window command prompt under windows terminal, the tab key behaviour for directory/file completion is inconsistent and does not circulate through all of the options

```none
Windows build number: [run `[Environment]::OSVersion` for powershell, or `ver` for cmd]
Windows Terminal version (if applicable):
Windows 10.0.19041.388  WT 1.1.2021.0
Any other software?
```

# Steps to reproduce

cd to a directory with multiple subdirectories all starting with the same letter
cd {firstletter} followed by tabbing through the directories.

# Expected behavior
All of the directory names with the prefix letter are cycled through with the tab key

# Actual behavior

The tab key cycles through alternate directories

**Comment by zadjii-msft** [comment]
You wouldn't happen to be using the workaround for "no keyboard", #4448? That's lead to some reports of the Tab key getting duplicated

**Comment by ogdenpm** [comment]
MikeI am. Seems like a problem either wayMark
    	
    	

    	RegardsMark Ogden
    
  




On Mon, Aug 3, 2020 at 2:20 PM +0100, "Mike Griese" <notifications@github.com> wrote:















You wouldn't happen to be using the workaround for "no keyboard", #4448? That's led to some reports of the Tab key getting duplicated

**Comment by zadjii-msft** [comment]
Hey sorry for the delayed reply. The old workaround for #4448 is what seems to cause the double-tab behavior, but _Good News Everyone_, it looks like there's a real fix for #4448 now - you need to re-enable the "Touch and Handwriting Service". See that thread for the latest details. Thanks!


/dup #4448
